manifest_staging/charts/secrets-store-csi-driver/templates/secrets-store-csi-driver-windows.yaml

```diff
@@ -28,6 +28,8 @@ spec:
 {{- end }}
     spec:
       serviceAccountName: secrets-store-csi-driver
+      imagePullSecrets:
```

Comment:
Wouldn't this need to be inside an if conditional? If not, this will add an empty `imagePullSecrets` in the yaml.

```bash
{{- if .Values.imagePullSecrets }}
  imagePullSecrets:
{{ toYaml .Values.imagePullSecrets | indent 8 }}
{{- end }}
```

Reply:
`imagePullSecrets` key isn't in the conditional. If there are no image pull secrets defined, then it'll add the empty `imagePullSecrets` key to the rendered yaml.

Reply:
Yes, somehow I lost the conditional, how I don't know